profiles/base/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM localhost/oci_env/pulp:base
+RUN echo "fastestmirror=1" >> /etc/dnf/dnf.conf
```

Comment:
To prevent additional unnecessary layer creation it would be better to combine this operation with the one that uses `dnf`.

Reply:
We don't have any direct dnf calls in this Dockerfile.